srcpkgs/certbot-apache/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'certbot-apache'
 pkgname=certbot-apache
-version=2.6.0
-revision=2
-build_style=python3-module
-hostmakedepends="python3-setuptools"
+version=2.7.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-poetry-core python3-setuptools python3-wheel"
+makedepends="python3-setuptools python3-wheel"
```

Comment:
Is this really necessary?

Reply:
The makedepends? I don't think so. I have built it without, I think I had it there while troubleshooting something and forgot. It's gone now.